Source/FieldSolver/WarpXPushFieldsEM.cpp

```diff
@@ -152,15 +159,22 @@ void WarpX::PSATDBackwardTransformEBavg (
 {
     const SpectralFieldIndex& Idx = spectral_solver_fp[0]->m_spectral_index;
 
+    // No need to fill guard cells for averaged fields
+    const amrex::IntVect& fill_guards = amrex::IntVect(0);
```

Comment:
Here I'm setting `fill_guards` so that the guard cells are **not** filled for the **averaged** E and B fields, since those are not damped when we apply "damped" boundary conditions (which is the only case where we currently want to fill the guard cells for inverse FFTs of the E and B fields). I could change this if we think it would more robust.

Reply:
Changed this in 844df7f: we now treat the averaged E and B fields in the same way as the regular E and B fields.